.gitignore

```diff
@@ -0,0 +1 @@
+.flutter_location_config
```

Comment:
This .gitignore is not necessary. The config file, if present, goes up one more level, just outside the repository. (It's designed that way so that a project that maps this project in as a dependency using a tool like gclient could check in the .flutter_location_config file as part of their containing repo to point to the location where the flutter tree is mapped in.)